src/components/ebay-inline-notice/index.marko

```diff
@@ -0,0 +1,14 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+$ var status = input.status || "attention";
+static var ignoredAttributes = ["status", "a11yHeadingText"];
+
+<ebay-notice-base
+    class=[`inline-notice--${status}`, input.class]
+    status=status
+    prefixClass="inline-notice"
+    root="div"
+    header-root="span"
+    main-root="span"
+    ...input
```

Comment:
Spreading the `input` at the bottom here will override the `class` we've set above.